strings/french.strings

```diff
@@ -1270,7 +1270,7 @@
 "Read more" = "Lire la suite";
 
 /* views/containers/proposals/ProposalBox.vue */
-"{strong_}You{_strong} are proposing:" = "{strong_}Vous{_strong} proposez:";
+"{strong_}You{_strong} are proposing:" = "{strong_}Vous{_strong} proposez :";
```

Comment:
Is it normal in French to have a space before colon?

Reply:
Oh OK, I see that apparently in fr-FR it is: https://github.com/okTurtles/group-income-simple/pull/958#discussion_r475359117